internal/server/authn/oidc_test.go

```diff
@@ -0,0 +1,14 @@
+package authn
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestValidateInvalidURL(t *testing.T) {
```

Comment:
Can't test an invalid client ID and client secret without relying on external URLs that could change, so skipped testing those for now.